ex03.ex

```diff
@@ -101,8 +120,21 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal([ h | t1],[ h | t2]) do
```

Comment:
Good job enumerating the cases.

There's a wee cheat you can use, too:

~~~ elixir
  def list_equal([], []),                              do: true
  def list_equal([ h | t1 ], [h | t2]),                do: list_equal(t1, t2)
  def list_equal(a, b) when is_list(a) and is_list(b), do: false
~~~